### DISCUSSION TITLE
Absolute collection binding

### DISCUSSION THREAD

**Discussion by sinsedrix** [dotnet/maui#9681] [Ideas]
Is there a way to bind a collection of items to an AbsoluteLayout ?

    <AbsoluteLayout>
        <AbsoluteLayout.Children ItemsSource="{Binding Bunch}" />
        
    </AbsoluteLayout>

I'd expect something like:

    <AbsoluteLayout.ItemTemplate>
        <DataTemplate x:DataType="{x:Type vm:ItemVm}">
            <Button AbsoluteLayout.LayoutBounds="{Binding LayoutBounds}" />
        </DataTemplate>
    </AbsoluteLayout.ItemTemplate>

CollectionView doesn't support an absolute item layout. Is there a way to implement one?

**Comment by sinsedrix** [comment]
Creating a test repo: https://github.com/sinsedrix/MauiAbsoluteCollection

**Comment by DoubleDBE** [comment]
https://docs.microsoft.com/en-us/dotnet/maui/user-interface/layouts/bindablelayout

**Reply by sinsedrix** [reply]
It doesn't seem to work:
> A value of type BindableLayout can't be added to a collection or dictionary of type IList`1'

**Reply by sinsedrix** [reply]
Ok, I found the right syntax and managed to make it work.
Test repo updated with page4.
